.yamllint.yml

```diff
@@ -0,0 +1,12 @@
+---
+extends: "default"
+rules:
+  comments: "enable"
+  empty-values: "disable"
+  indentation:
+    indent-sequences: "consistent"
+  line-length: "disable"
+  quoted-strings:
+    quote-type: "double"
+ignore: |
+  .venv/
```

Comment:
Is this for your local development? I see no tooling/CI changes that will look at this.

Reply:
I used it to validate the YAML I touched was good. Not adding to CI since it has a problem with a lot of other files and I don't want to scope-creep this PR. I'll remove it and perhaps do a separate PR if adding yamllint is wanted?

Reply:
> I used it to validate the YAML I touched was good. Not adding to CI since it has a problem with a lot of other files and I don't want to scope-creep this PR. I'll remove it and perhaps do a separate PR if adding yamllint is wanted?

This would be preferred, as at present it would require optional setup in development environments with no enforcement in CI.